sound/soc/sof/core.c

```diff
@@ -22,6 +22,7 @@
 int sof_core_debug;
 module_param_named(sof_debug, sof_core_debug, int, 0444);
 MODULE_PARM_DESC(sof_debug, "SOF core debug options (0x0 all off)");
+EXPORT_SYMBOL(sof_core_debug);
```

Comment:
not sure if this EXPORT is something we want to do? @kv2019i @ranj063 what do you think?

Reply:
> not sure if this EXPORT is something we want to do? @kv2019i @ranj063 what do you think?

This is needed in case we want to use the sof_core_debug from another module.

Reply:
@plbossart wondered about the same. Easy fix would be use a different parameter and put it under "intel/", but now that we use a generic debug flag, also the option should be here. The variable is in sof-priv.h and there are some examples of plain global variables exported (like sound_debugfs_root), so I guess this is ok.

Reply:
maybe mirror the parameter into a structure (sdev) and test the structure member?